state.json

```diff
@@ -233,7 +233,9 @@
                 "name": "pocket",
                 "random": true,
                 "commands": [
-                    "gslide id=1gDlOjkeMb9xLHzxhb5P5XL7gpIq08w0hdjCdykRIGTY comment=\"Pocket slideshow\""
+                    "gslide id=1gDlOjkeMb9xLHzxhb5P5XL7gpIq08w0hdjCdykRIGTY comment=\"Pocket slideshow\"",
+                    "https://forecast.io/embed/#lat=37.779&lon=-122.41499&name=Pocket%20-%20SanFrancisco&color=#DC143C&font=Arial zoom=300 comment=\"SF Weather\"",
+                    "https://modeanalytics.com/getpocket/reports/13a9c899c073?secret_key=accf7cf6cf7e55cc0c969079"
```

Comment:
Now that I can see this, I checked and unfortunately this dashboard won't work in Corsica. This site sets the HTTP header `x-frame-options: SAMEORIGIN`, which is incompatible with Corsica. This would result in a blank white screen on the Corsica displays.

Reply:
mode analytics also has links for embedding via static iframe or via javascript, any idea if those links might work?

Reply:
The links for embedding via static iframe should work. If not, then we could try and write some code that would use the JS option.

Reply:
okay, will try it. Mode does say it's an internal link for logged in users, but maybe it'll work since it's set to be a public url